programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -492,6 +496,116 @@ static void enroll(int argc, char *argv[]) {
     uv_run(l, UV_RUN_DEFAULT);
 }
 
+static tunnel_comand *cmd;
+
+static int dump_opts(int argc, char *argv[]) {
+    static struct option opts[] = {
+            {"identity", optional_argument, NULL, 'i'},
+            {"dump_path", optional_argument, NULL, 'p'},
+    };
+    int c, option_index, errors = 0;
+    optind = 0;
+
+    tunnel_ziti_dump *dump_options;
+    dump_options = calloc(1, sizeof(dump_options));
+    cmd = calloc(1, sizeof(tunnel_comand));
```

Comment:
when does cmd get freed?

Reply:
freed cmd in the on_connect function

Reply:
Ok, I see now that the `cmd` pointer is a static variable in this file and that it gets freed in on_connect. Now I question whether it should be freed? What happens to `cmd` when subsequent connections are made?